packages/core/src/components/Table/Table/Table.tsx

```diff
@@ -39,16 +40,6 @@ export interface ITableProps extends VibeComponentProps {
   withoutBorder?: boolean;
 }
 
-export interface ITableContext {
```

Comment:
Haven't we considered it as breaking if it's taken from dist?

Reply:
I'm not sure tbh
I do remember we've talked about it, but I don't think we need to cover those kind of cases
we do not export the `TableContext` component anyway outside from Vibe